appveyor.yml

```diff
@@ -0,0 +1,82 @@
+# Specify version format
+version: "{build}"
+
+# Operating system (build VM template)
+os: Visual Studio 2017
+
+# build platform, i.e. Win32 (instead of x86), x64, Any CPU. This setting is optional.
+platform:
+  - x64
+
+# Do not build feature branch with open Pull Request
+skip_branch_with_pr: true
+
+skip_commits:
+  files:
+    .travis.yml
+    .gitignore
+
+# specify custom environment variables
+environment:
+  MSVC_DEFAULT_OPTIONS: ON
+  RA_REPOSITORY_ROOT: C:\projects\Radium-Engine
+  RA_BUILD_DIR: C:\projects\Radium-Engine\build
+  RA_BUNDLE_DIR: C:\projects\Radium-Engine\Bundle-MSVC
+  RA_RELEASE_NAME: Radium-Engine-%platform%-%configuration%
```

Comment:
These absolute paths are going to be a problem for people who use this location

Reply:
Oh wait. Is it the CI script ?

Reply:
It's only for AppVeyor VMs, what's the problem ?
You are not supposed to use this script on your computer.

Reply:
OK. my bad.